srcpkgs/ocaml/template

```diff
@@ -1,26 +1,21 @@
 # Template file for 'ocaml'
 pkgname=ocaml
-version=4.14.0
-revision=2
+version=5.1.0
+revision=1
 build_style="gnu-configure"
 configure_args="--with-pic --libdir=/usr/lib/ocaml"
 make_build_target="world.opt"
-makedepends="ncurses-devel libX11-devel"
-short_desc="Main implementation of the Caml language"
+makedepends="libzstd-devel"
```

Comment:
I don’t know void’s package system’s syntax but from the error messages it looks like the library is not kept after installation (this is currently linked dynamically so it should be installed at runtime too)

Reply:
The problem is that it's also needed to be linked against.